src/hotspot/share/runtime/continuationFreezeThaw.cpp

```diff
@@ -675,7 +675,7 @@ void FreezeBase::freeze_fast_copy(stackChunkOop chunk, int chunk_start_sp CONT_J
   assert(!(_fast_freeze_size > 0) || _orig_chunk_sp - (chunk->start_address() + chunk_new_sp) == _fast_freeze_size, "");
 
   intptr_t* chunk_top = chunk->start_address() + chunk_new_sp;
-  assert(_empty || *(address*)(_orig_chunk_sp - frame::sender_sp_ret_address_offset()) == chunk->pc(), "");
+  assert(_empty || ContinuationHelper::return_address_at(_orig_chunk_sp - frame::sender_sp_ret_address_offset()) == chunk->pc(), "");
```

Comment:
This line is way too long too. 
```suggestion
  if (! _empty) {
    address *retaddr_slot = _orig_chunk_sp - frame::sender_sp_ret_address_offset();
    assert(|| ContinuationHelper::return_address_at(retaddr_slot) == chunk->pc(),
         "Saved return address is bad");
 }
```

Reply:
Thanks for your suggestion. Updated in the latest commit.

Note that `#ifdef ASSERT` is added in my commit, since variables `_empty` and `_orig_chunk_sp` belong to that scope.

Reply:
Please find some other extremely long lines in this patch. One is 138 characters long! Anything much more than 80 is a candidate for attention. It's very hard to read. And please don't use empty comment strings in assertions.
All of this is about code readability.

Reply:
Thanks for pointing this out.
Updated in the latest commit. Hope the code readability is improved now.